gouttelette/cmd/content_library_data.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
```

Comment:
This is more like a hack and I would prefer to keep it in the `vmare_rest_code_generator` for now.

Reply:
As discussed, this file will be moved to the right location after the generator is fully functional. For now, we have decided to keep this file in gouttelette.